dev/NavigationView/NavigationView.cpp

```diff
@@ -215,45 +265,43 @@ void NavigationView::OnApplyTemplate()
 
     m_topNavGrid.set(GetTemplateChildT<winrt::Grid>(c_topNavGrid, controlProtected));
 
+    UpdateNavigationViewItemsFactory();
+
     // Change code to NOT do this if we're in top nav mode, to prevent it from being realized:
-    if (auto leftNavListView = GetTemplateChildT<winrt::ListView>(c_menuItemsHost, controlProtected))
+    if (auto leftNavRepeater = GetTemplateChildT<winrt::ItemsRepeater>(c_menuItemsHost, controlProtected))
```

Comment:
>leftNavRepeater [](start = 13, length = 15)

The way this is set up means that m_leftNavRepeater will never be set to nullptr. Meaning you could hold onto a stale pointer.

Reply:
I'm not sure what you mean. m_leftNavRepeater gets set to nullptr in the beginning of the call to this function in `UnhookEventsAndClearFields`

---
In reply to: [353399778](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r353399778) [](ancestors = 353399778)

Reply:
I see, seems okay then.

---
In reply to: [354500162](https://github.com/microsoft/microsoft-ui-xaml/pull/1683#discussion_r354500162) [](ancestors = 354500162,353399778)